dialogflow/spec/create_delete_entity_spec.rb

```diff
@@ -0,0 +1,102 @@
+# Copyright 2018 Google, Inc
```

Comment:
The samples being tested are `entity_management.rb` so I expect this to be called `spec/entity_management_spec.rb`

Reply:
done